### ISSUE TITLE
Wrong death notification

### ISSUE THREAD

**Issue by lynxlynxlynx** [ja2-stracciatella/ja2-stracciatella#110]
Originally reported by: **JAsmine-ja2 (Bitbucket: [JAsmine-ja2](http://bitbucket.org/JAsmine-ja2), GitHub: [JAsmine-ja2](http://github.com/JAsmine-ja2))**

---

I recently started a new game on v0.12.258-win to test my upcoming mod for stracciatella. I received an email with a death notification for Scully:
![http://img.xrmb2.net/images/346858.jpeg](http://img.xrmb2.net/images/346858.jpeg)
However, Scully is still available, even after quiting and reloading the game or after comressing time. Noone died so far.
![http://img.xrmb2.net/images/832761.jpeg](http://img.xrmb2.net/images/832761.jpeg)

I have made some alterations to the game, but have definitely not messed with this, so I guess it's an stracciatella-issue.

Savegame is attached.

---
- Bitbucket: https://bitbucket.org/gennady/ja2-stracciatella/issue/110

**Comment by JAsmine-ja2** [comment]
Haven't encountered this bug in my latest (long-lasting) campaign, yet.
Stracc 0.15.1, JA2 1.12 English, Windows

**Comment by daviel** [comment]
So I tested this:
Every 24 ingame-hours there is a procedure running, calculating the chances of a merc to work for others than the player(can't be hired then). If a merc is working outside there will be another calculation for the chance of the merc to die based on his experience-level and social traits(like being psycho).

I tweaked the numbers so I would have a very high chance of mercs dying and getting the notification. Tested this about 50 times and could not once reproduce the state of receiving the email and still can hire the "dead" merc. 
Another thing to mention is that the only place this notification is possible in the whole codebase is in the 24h-procedure. So there can't be a sideeffect which calls this.

There a constants for email generation in the EMail.h I checked them for duplicates but didn't found any. The only thing I can come up with is that some of your mods used the constants in some way and leading for email-generation for a completely different events.

I think the modifications could have messed up with this. We can't tell for sure because we don't know what was changed by the issue-creator.
